system_tests/noxfile.py

```diff
@@ -379,8 +380,9 @@ def mtls_http(session):
     )
 
 
-@nox.session(python=PYTHON_VERSIONS_SYNC)
+@nox.session(python=PYTHON_VERSIONS_ASYNC)
 def external_accounts(session):
+    session.env[ALLOW_PLUGGABLE_ENV] = "1"
     session.install(
         *TEST_DEPENDENCIES_SYNC,
```

Comment:
Does this need to change as well?

Reply:
Yes, without this environment variable set, pluggable auth tests will fail

Reply:
Sorry, I meant the test dependencies, only `*TEST_DEPENDENCIES_SYNC` was being used and I assumed they would need to be updated to `*TEST_DEPENDENCIES_ASYNC` to reflect the nox session change

Reply:
Ah, gotcha.  I updated the nox session to use SYNC instead, because I believe Chuan updated the code to work with 2.7

Reply:
Seems like it [failed](https://source.cloud.google.com/results/invocations/2b166d53-6290-4153-a643-8a353f83a9ca/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fgoogle-auth-library-python%2Fpresubmit%2Fsystem-3.7/log) with 2.7 :( 

I don't have the context for this so I'm okay with moving it back to how you had it